packages/types/src/asset.ts

```diff
@@ -6,3 +6,17 @@ export type FindAssetsApiResponse = {
 };
 
 export type SampleAssetApiResponse = string;
+
+export const directories = {
+  users: {
+    images: "users/images/",
+  },
+  tutors: {
+    video: "tutors/videos/",
+    thumbnail: "tutors/thumbnails/",
+  },
+  reports: {
+    logs: "reports/logs",
+    screenshots: "reports/screenshots",
+  },
+} as const;
```

Comment:
Types pakcage should never has constants. It is jsut pure types. It can has enums only

Reply:
You can add it locally to the server constants. No other service/pkg is intertested in this information.

Reply:
Looks good with unions.